src/app/_components/SocketInitializer.tsx

```diff
@@ -39,8 +46,20 @@ const SocketInitializer = ({
     setChatRoomSelect: state.setChatRoomSelect,
   }));
 
+  const { setNewNotifications } = useNotificationsStore((state) => ({
+    setNewNotifications: state.setNewNotifications,
+  }));
+
   const queryClient = useQueryClient();
 
+  const getFilterOptionData = (option?: string) => {
+    const data = queryClient.getQueryData<INotificationsPagesData>([
+      'notifications',
+      option,
+    ]);
+    return data ? ['notifications', option] : false;
+  };
+
```

Comment:
```suggestion
  const getFilterOptionData = (option?: string) => {
    const hasData = !!queryClient.getQueryData<INotificationsPagesData>(['notifications', option]);
    return hasData ? ['notifications', option] : ['notifications'];
      'notifications',
      option,
    ]);
  };
```
일관된 반환 값 타입이 좋을거 같아요..! 하단에서도 
`const [ _, filterOptionData ] = getFilterOptionData(option);`
이런식으로 사용하면 될 듯합니당

Reply:
``` js
  const getFilterOptionData = (option?: string) => {
    const hasData = queryClient.getQueryData<INotificationsPagesData>([
      'notifications',
      option,
    ]);
    if (hasData) {
      return ['notifications', option];
    }
  };
```

해당 방식으로 변경 진행할게~